HotelPriceScout/Pages/Index.razor

```diff
@@ -7,178 +7,215 @@
+                            else if (item.Price < MarketPriceItem.Price)
+                            {
+                                <tr>
+                                    <td class="col text-center">@item.Name<span class="oi oi-arrow-right text-center"></span></td>
+                                    <td class="@(priceList.Last()==item ? "table-success rounded-bottom":"table-success")">@item.Price,-</td>
+                                </tr>
+                            }
+
+                            else
+                            {
+                                <tr>
+                                    <td class="col text-center"> @MarketPriceItem.Name<span class="oi oi-arrow-right text-center"></span></td>
+                                    <td class="table-info">@MarketPriceItem.Price,-</td>
+                                </tr>
+                            }
+                        }
+                    </table>
+                    }
             </div>
-        }
-    </div>
+            <div>Date: @Cal.DayClicked</div>
+        </div>}
+</div>
 
-@code {
+@code {int mPrice;
     public int AdultsPicked = 1;
     public class numberOfAdults
     {
         public int Adults { get; set; }
         public string Text { get; set; }
     }
+
     public List<numberOfAdults> Adults = new List<numberOfAdults>()
-        {
+    {
             new numberOfAdults(){ Adults= 1, Text= "1 adult" },
             new numberOfAdults(){ Adults= 2, Text= "2 adults" },
             new numberOfAdults(){ Adults= 3, Text= "4 adults" },
-        };
+    };
 
     Dashboard Cal = new Dashboard();
     IEnumerable<string> listOfHotels = Enumerable.Empty<string>();
 
     List<Prices> priceList;
     Prices MarketPriceItem;
+    IEnumerable<marketprice> CalenderMarket;
+
     private void GenerateThermometer()
     {
-        priceList =  PriceMeterGenerator.PriceListGenerator(); // This needs to get its data from the database, as a list of Price objects
-        MarketPriceItem =  (PriceMeterGenerator.MarketFinder(priceList));
+        priceList = PriceMeterGenerator.PriceListGenerator(); // This needs to get its data from the database, as a list of Price objects
+        MarketPriceItem = (PriceMeterGenerator.MarketFinder(priceList));
         priceList.Sort();
     }
 
+    public async void OnClickUpdate()
+    {
+        CalenderMarket = await (Cal.DisplayComaredPrices(Cal.toDay.ToString("yyyy-MM-dd"), Cal.monthEnd.ToString("yyyy-MM-dd")));
+    }
+
     protected override async Task OnInitializedAsync()
     {
         GenerateThermometer();
-        listOfHotels = await _db.LoadStaticHotelResources("total");
+
         Cal.CreateMonth();
 
-       
+        CalenderMarket = await (Cal.DisplayComaredPrices(Cal.tempDate.ToString("yyyy-MM-dd"), Cal.monthEnd.ToString("yyyy-MM-dd")));
+    }
+
+    public int MarketPriceDaySetter(int k)
+    {
+        if (new DateTime(Cal.year, Cal.month, k, 23, 59, 59) >= Cal.toDay)
```

Comment:
Hvis i gerne vil sikre, at i får kl. 00:00:00, så er der en method som hedder .Date på DateTime.

Reply:
vi vil sikre os at klokken på dagen er senere end det den er på hans computer når han checker så der for 23:59:59 for ellers viser den ikke daten for den dag han er på.

Reply:
Makes sense. Skriv en comment i koden om det, for det er ikke tydeligt, når man læser det.